Delicate-Butterflies/Bangazon-Site
# Product nav drop jr

Related Issue:
#12 

Description:
add dropdown to nav that shows all product types, as well as categories link and latest link
-also reworked nav template to include all links

Chai/Mocha Tests File:

PR Testing Steps:
- pull, npm start, look at differences between logged out/in states
- check functionality

Check list before you make a PR:
